src/cdk/drag-drop/drag-events.ts

```diff
@@ -67,6 +67,8 @@ export interface CdkDragDrop<T, O = T> {
   isPointerOverContainer: boolean;
   /** Distance in pixels that the user has dragged since the drag sequence started. */
   distance: {x: number, y: number};
+  /** Point where the mouse was when the item was dropped */
```

Comment:
This should refer to "point", rather than "mouse", because we also support touch screens.

Reply:
Thanks for the Input. I also thisnk dropPoint was not the best name. I remaned it to dropPosition.

Reply:
I think that the `dropPoint` name was better than `dropPosition` since it is a 1x1 point at which the element was dropped. My comment above was referring to the doc string which should talk about "pointer" instead of "mouse".

Reply:
Ok, you are right. Reverted it back to `dropPoint`